runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -80,6 +80,21 @@ static struct fid_cq* ofi_rxCQ;         // receive endpoint CQ
 static struct fid_av* ofi_av;           // address vector, table style
 static fi_addr_t* ofi_rxAddrs;          // remote receive addrs
 
+//
+// For performance reasons, we use manual progress.  We direct RMA
```

Comment:
>  For performance reasons, we use manual progress.

Is this comment still accurate?

Reply:
Yeah, not entirely.  When I added defaulting to auto progress for sockets I missed updating this.  Will do so now.